wasm/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "css-inline-wasm"
+version = "0.3.0"
+authors = ["Dmitry Dygalo <dadygalo@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+name = "css_inline"
+crate-type = ["cdylib"]
+
+[build-dependencies]
```

Comment:
At the moment there is no usage of `built`, so I propose to remove it. In the Python bindings, it is used to provide some meta-information about the Rust side (all dependencies, environment, etc), but here I am not sure if there are any standard ways to provide such info (I'll be happy to apply them if there are)